test/lit/passes/signature-refining.wast

```diff
@@ -490,3 +490,137 @@
     )
   )
 )
+
+(module
+  ;; CHECK:      (type $struct (struct_subtype  data))
+  (type $struct (struct_subtype data))
```

Comment:
Is there a doc that defines these new syntax? (`struct_subtype`, `func_subtype`, ...) It sounds like this `struct` is a subtype of `data`, but this doesn't specify any more fields. Does that mean `struct` is the same as `data`?

Reply:
I think this is the doc: https://docs.google.com/document/d/1DklC3qVuOdLHSXB5UXghM_syCh-4cMinQ50ICiXnK3Q/edit#

`(type $struct (struct_subtype data))` means a struct with no fields. Not very interesting but it is a unique nominal type which is enough for some tests.